lib/pwa.js

```diff
@@ -82,6 +83,53 @@ exports.list = function(skip, limit, sort) {
 };
 
 /**
+ * Add all PWAs from the DB into the text search index.
+ */
+exports.indexAllPwas = _ => {
```

Comment:
It's a good idea, I think I rather leave it here for now, it's very related to PWA.